op-conductor/conductor/service.go

```diff
@@ -584,15 +586,19 @@ func (oc *OpConductor) startSequencer() error {
 
 		if uint64(unsafeInCons.ExecutionPayload.BlockNumber)-unsafeInNode.NumberU64() == 1 {
 			// tries to post the unsafe head to op-node when head is only 1 block behind (most likely due to gossip delay)
-			if err = oc.ctrl.PostUnsafePayload(context.Background(), unsafeInCons); err != nil {
+			if err = oc.ctrl.PostUnsafePayload(ctx, unsafeInCons); err != nil {
 				oc.log.Error("failed to post unsafe head payload envelope to op-node", "err", err)
 			}
 		}
 		return ErrUnsafeHeadMismarch // return error to allow retry
 	}
 
-	if err := oc.ctrl.StartSequencer(context.Background(), unsafeInCons.ExecutionPayload.BlockHash); err != nil {
-		return errors.Wrap(err, "failed to start sequencer")
+	if err = oc.ctrl.StartSequencer(ctx, unsafeInCons.ExecutionPayload.BlockHash); err != nil {
+		if !strings.Contains(err.Error(), driver.ErrSequencerAlreadyStarted.Error()) {
```

Comment:
Can we just use:
```suggestion
		if !errors.Is(err, driver.ErrSequencerAlreadyStarted) {
```

Probably can't because I think `oc.ctrl` is actually making a HTTP request so we're losing the error type.  That's probably worth a comment though or someone's likely to change it (or even better a unit test).

Reply:
You're right, we cannot directly compare as the result is returned from op-node JSON RPC server.

I've added the comment mentioning that, and regarding tests, will have one in e2e tests, currently e2e tests is flaky (I have another PR to fix this), will add it once it's ready.